tests/TestCase.php

```diff
@@ -44,11 +44,15 @@ protected function setUpDatabase()
         $this->app['db']->connection()->getSchemaBuilder()->create('dummies', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
+            $table->string('custom_column_sort');
             $table->integer('order_column');
         });
 
         collect(range(1, 20))->each(function (int $i) {
-            Dummy::create(['name' => $i]);
+            Dummy::create([
+                'name' => $i,
+                'custom_column_sort' => chr(123 - $i), // backwards from z
```

Comment:
What is this voodoo? 😀 Can it be written in a more expressive way?

Reply:
The goal was to have values that were not sorted the same way as the name/index/default sort, so I chose to enumerate characters backwards from the letter `z`. I could also have picked negative numbers and used `0 - $i`. Or, I could also generate a random number with `rand()`. Do you have a preference or alternate suggestion?

Reply:
Let's just go with `rand`, that's a bit more expressive in this situation.